test/e2e/shim-install.sh

```diff
@@ -6,22 +6,20 @@ set -ex
 
 # Build gvisor-containerd-shim
 if [ "${INSTALL_LATEST}" === "1" ]; then
-{ # Step 1: Download gvisor-containerd-shim
+{ # Step 1(release): Install gvisor-containerd-shim
 LATEST_RELEASE=$(wget -qO - https://api.github.com/repos/google/gvisor-containerd-shim/releases | grep -oP '(?<="browser_download_url": ")https://[^"]*' | head -1)
 wget -O gvisor-containerd-shim
 chmod +x gvisor-containerd-shim
+sudo mv gvisor-containerd-shim /usr/local/bin/gvisor-containerd-shim
 }
 else
+{ # Step 1(dev): Build and install gvisor-containerd-shim and containerd-shim-runsc-v1
     make
```

Comment:
Is there a good way to check as part of the test that it's actually using the v2 shim as opposed to the v1 without just trusting that the config.toml is correct?

Maybe just something like this?

    ps aux | grep containerd-shim-runsc-v1

Not sure it needs to be in the quick start, but it would be nice to have as part of the test script.

Reply:
Done